plugins/goodreads.koplugin/doublekeyvaluepage.lua

```diff
@@ -116,44 +120,37 @@ function DoubleKeyValueItem:init()
     end
     local key_w = RenderText:sizeUtf8Text(0, self.width, self.cface_down, self.key).x
     local value_w = RenderText:sizeUtf8Text(0, self.width, self.cface_up, self.value).x
-    if key_w > self.width then
-        self.show_key = RenderText:truncateTextByWidth(self.key, self.cface_down, self.width)
+    if key_w > self.width - 2*padding then
+        self.show_key = RenderText:truncateTextByWidth(self.key, self.cface_down, self.width - 2*padding)
     else
         self.show_key = self.key
     end
-    if value_w > self.width then
-        self.show_value = RenderText:truncateTextByWidth(self.value, self.cface_up, self.width)
+    if value_w > self.width - 2*padding then
+        self.show_value = RenderText:truncateTextByWidth(self.value, self.cface_up, self.width - 2*padding)
     else
         self.show_value = self.value
     end
     local h = self.dimen.h / 2
     local w = self.dimen.w
     self[1] = FrameContainer:new{
-        padding = Screen:scaleBySize(10),
+        padding = padding,
         bordersize = 0,
+        width = self.width,
+        height = self.height,
         VerticalGroup:new{
-            dimen = Geom:new{ h = h, w = w },
-            padding = Screen:scaleBySize(10),
             LeftContainer:new{
-                padding = Screen:scaleBySize(10),
-                dimen = Geom:new{ h = h, w = w },
+                padding = 0,
+                dimen = Geom:new{ h = h   , w = w },
```

Comment:
Btw, why'd you change this from what it was? (Space-wise) Same a few lines down.